source/administration/changelog.md

```diff
@@ -2,6 +2,215 @@
 
 This changelog summarizes updates to [Mattermost Team Edition](http://www.mattermost.org/), an open source team messaging solution released monthly under an MIT license, and [Mattermost Enterprise Edition](https://about.mattermost.com/pricing/), a commercial upgrade offering enterprise messaging for large organizations.
 
+## Release v3.2.0
+
+Expected release date: 2016-07-16
+
+### Highlights
+
+#### Languages
+
+- Added German translation for the user interface if enabled by the System Admin from **System Console > Localization > Available Languages**.
+
+#### Policy (Enterprise)
+
+- Restrict the permission levels required to send team invitiations in **System Console** > **Policy**.
+- Restrict the permission levels required to manage public and private channels, including creating, deleting, renaming, and setting the channel header or purpose. 
+
+#### SAML Single-Sign-On (Enterprise):
+
+- Users can sign in to Mattermost with their SAML credentials and new Mattermost user accounts are automatically created on first login. Mattermost pulls user information from SAML, including first and last name, email and username.
+- Mattermost officially supports Okta and Microsoft ADFS as the identity providers (IDPs), but you may also try configuring SAML for a custom IDP.
+
+#### Custom Emoji
+
+- Create Custom Emoji from the **Main Menu** > **Custom Emoji** when enabled from **System Console** > **Customization** > **Custom Emoji**.
+- Restrict the permissions required to create Custom Emoji (Enterprise).
+
+#### Performance
+- Gzip compression for static content files decreases time for first page load, enabled from **System Console** > **Configuration**.
+- Reduced the total Mattermost package size from 25.7MB to 18.9MB.
+
+### Improvements
+
+**Users**
+
+- After account creation, users are automatically directed to the team where they were invited instead of the Team Selection page.
+- Users can now be removed from teams via the **Main Menu** > **Manage Members** modal.
+
+
+**System Console**
+
+- Updated labeling of System Console settings in the UI for consistency and accuracy.
+- (Enterprise) Help text was added to the **Reload Configuration from Disk** and **Recycle Database Connections** buttons to explain their use in the switching databases with no down time.
+- Allow System Admins to create teams even if team creation is disabled via the System Console.
+
+**Notifications**
+
+- Address displayed in the email notification footer is now configurable in the System Console.
+- Direct Message desktop notifications now display with a "Direct Message" title.
+
+**Web UI**
```

Comment:
Thought of another one - not sure if it fits in web ui?

@mention autocomplete now filters on nickname, full name, and username